libs/vre/3rd-party-services/open-api/jest.config.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable */
 export default {
   displayName: 'vre-open-api',
-  preset: '../../../jest.preset.js',
+  preset: '../../../../jest.preset.js',
   setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
-  coverageDirectory: '../../../coverage/libs/vre/3rd-party-services/open-api',
+  coverageDirectory: '../../../../coverage/libs/vre/3rd-party-services/open-api',
```

Comment:
this I cannot say anything about. Looks unrelated to OpenAPI

Reply:
This is in the `open-api` directory and was necessary fix to get test passing but I will actually remove it from unit test configuration.